### DISCUSSION TITLE
Default file icons size

### DISCUSSION THREAD

**Discussion by jabrugger** [doublecmd/doublecmd#1115] [Suggestions]
### Description

Every time I install Double Commander I do at least two things: I reduce the size of the icons in the file panel, and I enable the option to show hidden/system files.
So I suggest that the standard size of the file panel icons be the same as the default size of the disk panel icons. The current default size of the file panel icons leaves too much white space between each file.
Thanks for considering this suggestion of improvement!

**Comment by alexx2000** [comment]
Icon size 16x16 is too small with modern full-hd displays. DC looks terrible with such small icons.

**Comment by jabrugger** [comment]
1920x1080:
Default icon size:
![imagen](https://github.com/doublecmd/doublecmd/assets/21174632/48bb5365-3529-48be-9336-bbcdbb4f3ca1)
"Corrected" icon size:
![imagen](https://github.com/doublecmd/doublecmd/assets/21174632/71266f6d-ab96-47df-84b4-b41c532ccb77)
Total Commander default icon size:
![imagen](https://github.com/doublecmd/doublecmd/assets/21174632/1acfb847-0364-43c9-a297-5737e55324ea)
A more dense file list looks better to me ... 
Maybe some options like this one could be selected during install or at first run?
